pytest.ini

```diff
@@ -21,7 +21,7 @@ python_functions =
 # always run in parallel (requires pytest-xdist, see test-requirements.txt)
 # and enable strict mode: require all markers
 # to be defined and raise on invalid config values
-addopts = -nauto --strict-markers --strict-config
+addopts = -nauto --strict-markers --strict-config --color=yes
```

Comment:
Would it be possible to instead add this flag in workflow config? Changing this file will affect local testing as well.

Reply:
Okay, I've added `PYTEST_ADDOPTS` to the workflow file and verified that it cooperates with the `addopts` config in `pytest.ini`.

Reply:
ergh, that isn't working.....